script/upgrades/MU01/governance/Config.sol

```diff
@@ -0,0 +1,357 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// solhint-disable func-name-mixedcase, contract-name-camelcase, function-max-lines, var-name-mixedcase
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+
+import { FixidityLib } from "mento-core/contracts/common/FixidityLib.sol";
+import { console2 as console } from "forge-std/Script.sol";
+
+import { Chain } from "script/utils/Chain.sol";
+import { Contracts } from "script/utils/Contracts.sol";
+import { Arrays } from "script/utils/Arrays.sol";
+
+library Config {
+  using FixidityLib for FixidityLib.Fraction;
+  using Contracts for Contracts.Cache;
+
+  uint8 private constant L0 = 1; // 0b001 Limit0
+  uint8 private constant L1 = 2; // 0b010 Limit1
+  uint8 private constant LG = 4; // 0b100 LimitGlobal
```

Comment:
is 4 on purpose, not 3 like consecutive numbers?

Reply:
Yeah, these are flags, think of them as switches that can be on or off, and which can be encoded into a single number, and thus a single variable.
```
L0             = 0b001; // in binary
L1              = 0b010;
LG             = 0b100;
L0_L1        = 0b011; // L0 | L1 == L0 + L1
L0_L1_LG = 0b111;  // L0 | L1 | LG == L0 + L1 + LG
```